web/src/App.js

```diff
@@ -61,6 +62,9 @@ class App extends React.Component {
             <Route path="/events" component={() =>
               <Events accessToken={this.state.accessToken} events={this.state.events} />
             } />
+            <Route path="/voting-contest" render={() =>
```

Comment:
I think we could simplify this to the `<Route path="/voting-contest" component={VotingContest} />` style route